docs/guide/replacement-guides/mkdirp.md

```diff
@@ -0,0 +1,59 @@
+---
+description: Modern alternatives to the mkdirp and make-dir packages for recursively creating directories in Node.js
+---
+
+# Replacements for `mkdirp` / `make-dir`
+
+# Alternatives
+
+## Node.js (since v10.12.0)
+
+Node.js v10.12.0 and up supports the `recursive` option in the [`fs.mkdir`](https://nodejs.org/api/fs.html#fsmkdirpath-options-callback) function, which allows parent directories to be created automatically:
+
+```js
```

Comment:
Maybe this first example isn't necessary? Since we show the diff afterwards which is far more useful I think